app/config/CorporationTaxHeaderCarrierForPartialsConverter.scala

```diff
@@ -17,7 +17,6 @@
 package config
 
 import javax.inject.{Inject, Singleton}
-
 import uk.gov.hmrc.crypto.PlainText
 import uk.gov.hmrc.play.bootstrap.filters.frontend.crypto.SessionCookieCrypto
 import uk.gov.hmrc.play.partials.HeaderCarrierForPartialsConverter
```

Comment:
Acceptance Tests fail due to the service info partial not being present,
There was the same issue on VAT that was resolved [here](https://github.com/hmrc/vat-frontend/pull/136/commits/d68049bc971ce423730cf5304fc062ad256c8c7b)

![Screenshot 2020-04-30 at 11 34 29](https://user-images.githubusercontent.com/6122384/80701375-34672c80-8ad7-11ea-9bd6-581b927a40ee.png)

Reply:
Thanks! I knew I needed to do this but clearly didn't get around to it
This is also the fix for add-tax-frontend that is also in progress.
I'll use `identity` rather than `cookie => cookie` though...